class-wc-facebookcommerce.php

```diff
@@ -201,6 +204,28 @@ public function add_wordpress_integration() {
 		/** Getter methods ********************************************************************************************/
 
 
+		/**
+		 * Gets the API instance.
+		 *
+		 * @since 2.0.0-dev.1
+		 *
+		 * @return SkyVerge\WooCommerce\Facebook\API
+		 */
+		public function get_api() {
+
+			if ( ! is_object( $this->api ) ) {
+
+				require_once __DIR__ . '/includes/API.php';
+				require_once __DIR__ . '/includes/API/Request.php';
+				require_once __DIR__ . '/includes/API/Response.php';
+
+				$this->api = new SkyVerge\WooCommerce\Facebook\API( $this->get_connection_handler()->get_access_token() );
```

Comment:
Should we verify if the token is available before trying to instantiate the API?

Reply:
I'm not against it. I didn't add a verification because we a lot of checks to bail if we don't have an access token, so is less likely that we try to use the API with a token.

Also, if the access token is not set, trying to use any of the API methods should throw a `SV_WC_API_Exception`, that we will likely try to catching whenever we use the API, so I thought that was good enough.

Do you think it would be better to check whether the plugin is connected here and throw `SV_WC_API_Exception` if it isn't ?

Reply:
I believe so, there is no reason to instantiate an object if nobody will be able to use it, IMO. It's probably better to throw the exception first and save the resources.

Reply:
Sounds good. I added the exception in 515ed07 and a new test in d933277.